backend/docs/Config.md

```diff
@@ -0,0 +1,251 @@
+#
+secrets:
+  ## keys ##
+  #
+  # A secret that is used to sign and verify session JWTs. The first item is used for signing. The whole list is used for verifying session JWTs.
+  #
+  keys:
+    - "change-me"
+session:
+  ## lifespan ##
+  #
+  # Sets how long a session JWT is valid.
+  #
+  # Default value: 1h
+  #
+  # Examples:
+  # - 1h
+  # - 10m
+  # - 720h
+  # - 15h115m
+  #
+  lifespan: "1h"
+  cookie:
+    ## domain ##
+    #
+    # Sets on which domain the cookie will be bound. Does not work cross domain, but only for subdomains.
+    #
+    domain: ""
+    ## http_only ##
+    #
+    # Sets whether the cookie is a http only cookie or can be read by javascript.
+    #
+    # Default value: true
+    #
+    http_only: true
+    ## same_site ##
+    #
+    # Configures the same site attribute of the session cookie.
+    #
+    # Default value: strict
+    #
+    # One of:
+    # - strict
+    # - lax
+    # - none
+    #
+    same_site: "strict"
+    ## secure ##
+    #
+    # Sets whether the cookie can only be read on secure sites.
+    #
+    # Default value: true
+    #
+    secure: true
+password:
+  ## enabled ##
+  #
+  # Configures whether password are enabled or not.
+  #
+  # Default value: false
+  #
+  enabled: false
+passcode:
+  ## ttl ##
+  #
+  # Sets how long a passcode is valid. Value is in seconds.
+  #
+  # Default value: 300
+  #
+  ttl: 0
+  email:
+    ## from_address ##
+    #
+    # Configures the sender of emails sent to the users.
+    #
+    from_address: ""
+    ## from_name ##
+    #
+    # Configures the sender name of emails sent to the users.
+    #
+    from_name: ""
+  ## smtp ##
+  #
+  # Configures the backend which smtp server will be used to sent emails.
+  #
+  smtp:
+    host: ""
+    ## port ##
+    #
+    # TODO:
+    #
+    # Default: 465
+    #
+    port: ""
+    user: ""
+    password: ""
+## webauthn ##
+#
+# Configures web authentication
+#
+webauthn:
+  ## timeout ##
+  #
+  # Configures how long a web authentication request is valid and the user can confirm it. Value is in milliseconds
+  #
+  # Default: 60000
+  #
+  timeout: 0
+  relying_party:
+    ## id ##
+    #
+    # ID sets the host on which web authentication can be used.
+    #
+    # Examples:
+    # - localhost
+    # - example.com
+    # - subdomain.example.com
+    #
+    id: "localhost"
+    ## display_name ##
+    #
+    # Sets the name which the web authentication authenticator will show during the ceremony
+    #
+    display_name: ""
+    ## origin ##
+    #
+    # Sets the origin of which web authentication can be used.
+    #
+    # Example:
+    # - http://localhost
+    # - https://example.com
+    # - https://subdomain.example.com
+    #
+    origin: "http://localhost"
+```
+
+## Explanation
+
+### Web Authentication
+
+Configuring Web Authentication might seem challenging, but is pretty straight forward.
```

Comment:
I don't understand sentences like this. I don't need them in a README. Depending on the person and the topic at hand anything might seem challenging ...